docs/src/pages/docs/functions/combinators.md

```diff
@@ -48,6 +48,55 @@ This is a very handy dandy function, used a lot. Pass it any value and it will
 give you back a function that will return that same value no matter what you
 pass it.
 
+#### converge
+
+`crocks/combinators/converge`
+
+```haskell
+converge :: (b -> c -> d) -> (a -> b) -> (a -> c) -> a -> d
+```
+
+Provides a means of passing an acculumating function and two branching functions.
+A value can be applied to the resulting function which will then be applied to
+each branching function, the results of which will be applied to the accumulating
+function.
+
+```javascript
+import { Just } from 'crocks/Maybe'
+import { alt } from 'crocks/pointfree'
+import { converge } from 'crocks/combinators'
+import { prop } from 'crocks/helpers'
+
+const divide = x => y => x / y
+const sum = xs => xs.reduce((m, n) => m + n, 0)
+const length = xs => xs.length
+
+converge(divide, sum, length)([1, 2, 3, 4, 5])
```

Comment:
it may be beneficial to create an `average` function for the benifiet of the siggy like

```js
// data :: [ Number ]
const data =
  [ 1, 2, 3, 4, 5 ]

// average :: [ Number ] -> Number
const average =
  converge(divide, sum, length)

average(data)
//=> 3
```

Reply:
I didn't declare variables/signatures for the objects in the second half of the example because it seemed too verbose. I can do it if that's desired, though.